### PR TITLE
docs: Update docstring of modules

### PR DESCRIPTION
**Pull request by LeeDongGeon1996**
### Description

 - Update docstring of modules.

### Checklist

- [x] My code follows the existing style, code structure, and naming taxonomy
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code and included any verifying manual calculations
- [x] I have added or updated unit tests that prove my fix is effective or that my feature works, and achieves sufficient code coverage.  New and existing unit tests pass locally and in the build (below) with my changes
- [x] My changes generate no new warnings and running code analysis does not produce any issues
- [x] I have added or run the performance tests that depict optimal execution times
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by LeeDongGeon1996** [comment]
@DaveSkender  Could you check if there's weird expression?

File: stock_indicators/_cslib/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Skender.Stock.Indicators
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-This module loads `Skender.Stock.Indicators.dll`, which is a compiled library package
+This module loads `Skender.Stock.Indicators.dll`(v1.22.4), which is a compiled library package
 from <https://github.com/DaveSkender/Stock.Indicators>, written in C#.
 
 It is currently using `.NET Standard 2.0`.
```
Comment (DaveSkender) [inline stock_indicators/_cslib/__init__.py:8]: Does Python.NET 3.x support newer .NET versions that are 2-3x faster than .NET Standard?

**Review by DaveSkender — APPROVED** [review]

File: stock_indicators/_cslib/__init__.py (hunk 1 above)
Comment (LeeDongGeon1996) [reply stock_indicators/_cslib/__init__.py:8]: Yeah, not now, but it will support .NET 6.0 in pythonnet v3.0.

Comment (LeeDongGeon1996) [reply stock_indicators/_cslib/__init__.py:8]: .NET Standard 2.1 is supported with current pythonnet version. Is it better than 2.0, right?

Comment (DaveSkender) [reply stock_indicators/_cslib/__init__.py:8]: Yah, I’d use the newest possible, they’re progressively faster, especially when you get to .NET Core versions.

Comment (LeeDongGeon1996) [reply stock_indicators/_cslib/__init__.py:8]: Okay, then I'll change the dll to .NET Standard 2.1.

Comment (DaveSkender) [reply stock_indicators/_cslib/__init__.py:8]: Yes, .NET Standard is a "bridge" standard between .NET Framework (older) and .NET Core (newer); it has the broadest compatibility between those two worlds.  Each progressive version is measurably faster.

Comment (LeeDongGeon1996) [reply stock_indicators/_cslib/__init__.py:8]: Ah, thank you for letting me know :)

**Pull request merged**